### ISSUE TITLE
focal_loss with softmax

### ISSUE THREAD

**Issue by czjghost** [vandit15/Class-balanced-loss-pytorch#20]
does someone has implement focal_loss with softmax correctly？could you recommend its link for me？thank you very much
